### PR TITLE
Added 3 sorting techniques and 1 String program.

### PR DESCRIPTION
**Pull request by karanS08**

### CODE REVIEW

**Comment by Souvik-Nayak** [comment]
The sorting techniques can be done as user input which will show result for different inputs !

**Comment by fms12** [comment]
add a comments for a better understanding

File: Searching & sorting/bubble_sort.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Bubble sort
+def bubble_sort(list):
+    for i in range(len(list)):
```
Comment (fms12) [inline Searching & sorting/bubble_sort.py:3]: add comments for explanation and make separate folder for sorting then try to push

Comment (fms12) [inline Searching & sorting/bubble_sort.py:3]: add comments for explanation and make separate folder for sorting then try to push

File: Searching & sorting/insertion_sort.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Insertion sort 
+def insertionSort(arr): 
+    for i in range(1, len(arr)): 
+        key = arr[i] 
+        j = i-1
+        while j >=0 and key < arr[j] : 
+                arr[j+1] = arr[j]
```
Comment (fms12) [inline Searching & sorting/insertion_sort.py:7]: add comments

File: Searching & sorting/selection_sort.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Selecion sort 
+
+def selection_sort(arr):
+    for i in range(len(arr)):
+        min_index = i
+        for j in range(i+1, len(arr)):
+            if arr[min_index] > arr[j]:
```
Comment (fms12) [inline Searching & sorting/selection_sort.py:7]: add comments

File: String/Shuffle_string.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# write a program to shuffle all the strings of a input word
+import random
+def shuffle(word):
+    word = list(word)
+    random.shuffle(word)
```
Comment (fms12) [inline String/Shuffle_string.py:5]: add comments

**Review by fms12 — CHANGES_REQUESTED** [review]

**Comment by karanS08** [comment]
@fms12  Please check now for any error.

**Comment by karanS08** [comment]
@fms12 Check now .

**Review by fms12 — APPROVED** [review]

**Pull request merged**